app/assets/javascripts/discourse/app/controllers/sidebar-section-form.js

```diff
@@ -95,7 +95,10 @@ class SectionLink {
   }
 
   #validInternal() {
-    return this.router.recognize(this.path).name !== "unknown";
+    return (
+      this.router.recognize(this.path).name !== "unknown" ||
+      this.path.match(/^\/my\/[a-z_\-\/]+$/)
```

Comment:
Perhaps we can extract the regexp to a constant so that the name of the constant will provide some hint about what the regexp is meant for?